cleanlab/object_detection/summary.py

```diff
@@ -34,6 +42,184 @@
+
+    predictions :
+        Predictions output by a trained object detection model.
+        For the most accurate results, predictions should be out-of-sample to avoid overfitting, eg. obtained via :ref:`cross-validation <pred_probs_cross_val>`.
+        This is a list of ``N`` ``np.ndarray`` such that ``predictions[i]`` corresponds to the model prediction for the `i`-th image.
+        For each possible class ``k`` in 0, 1, ..., K-1: ``predictions[i][k]`` is a ``np.ndarray`` of shape ``(M,5)``,
+        where ``M`` is the number of predicted bounding boxes for class ``k``. Here the five columns correspond to ``[x1,y1,x2,y2,pred_prob]``,
+        where ``[x1,y1,x2,y2]`` are coordinates of the bounding box predicted by the model
+        and ``pred_prob`` is the model's confidence in the predicted class label for this bounding box.
+
+        Note: Here, ``[x1,y1]`` corresponds to the coordinates of the bottom-left corner of the bounding box, while ``[x2,y2]`` corresponds to the coordinates of the top-right corner of the bounding box. The last column, pred_prob, represents the predicted probability that the bounding box contains an object of the class k.
+
+        For more information see the `MMDetection package <https://github.com/open-mmlab/mmdetection>`_ for an example object detection library that outputs predictions in the correct format.
+
+    auxiliary_inputs: optional
+        Auxiliary inputs to be used in the computation of counts.
+        The `auxiliary_inputs` can be computed using :py:func:`rank._get_valid_inputs_for_compute_scores <cleanlab.object_detection.rank._get_valid_inputs_for_compute_scores>`.
+        It is internally computed from the given `labels` and `predictions`.
+
+    Returns
+    -------
+    object_counts: Tuple[List, List]
+        A tuple containing two lists. The first is an array of shape ``(N,)`` containing the number of annotated objects for each image in the dataset.
+        The second is an array of shape ``(N,)`` containing the number of predicted objects for each image in the dataset.
+    """
+    if auxiliary_inputs is None:
+        auxiliary_inputs = _get_valid_inputs_for_compute_scores(ALPHA, labels, predictions)
+    return (
+        [len(sample["lab_bboxes"]) for sample in auxiliary_inputs],
+        [len(sample["pred_bboxes"]) for sample in auxiliary_inputs],
+    )
+
+
+def bounding_box_size_distribution(
+    labels=None,
+    predictions=None,
+    *,
+    auxiliary_inputs=None,
+    class_names: Optional[Dict[Any, Any]] = None,
+) -> Tuple[Dict[Any, List], Dict[Any, List]]:
+    """Return the distribution over sizes of annotated and predicted bounding boxes across the dataset, broken down by each class.
+
+    This method can help you find annotated/predicted boxes for a particular class that are abnormally big/small.
+
+    Parameters
+    ----------
+    labels:
```

Comment:
Can you hyperlink the format of labels/predictions/auxillary inputs for all functions but one in this page similarly to what is done [here](https://github.com/cleanlab/cleanlab/blob/6d45971cf83ccc85ac31d2591c54ce6b52a27281/cleanlab/object_detection/rank.py#L648)? (This way if we change anything in the format the update will only have to happen once per file)

I suggest linking labels/predictions definitions of `bounding_box_size_distribution` and `class_label_distribution` to the descriptions written out in `object_counts_per_image` since that appears first.